Lib/glyphsLib/classes.py

```diff
@@ -4279,39 +4313,60 @@ def gridLength(self):
     EMPTY_KERNING_VALUE = (1 << 63) - 1  # As per the documentation
 
     def kerningForPair(self, fontMasterId, leftKey, rightKey, direction=LTR):
-        # TODO: (jany) understand and use the direction parameter
-        if not self._kerning:
+        if direction == LTR:
+            kerntable = self._kerningLTR
+        elif direction == RTL:
+            kerntable = self._kerningRTL
+        else:
+            kerntable = self._kerningVertical
+        if not kerntable:
             return self.EMPTY_KERNING_VALUE
         try:
-            return self._kerning[fontMasterId][leftKey][rightKey]
+            return kerntable[fontMasterId][leftKey][rightKey]
         except KeyError:
             return self.EMPTY_KERNING_VALUE
 
     def setKerningForPair(self, fontMasterId, leftKey, rightKey, value, direction=LTR):
         # TODO: (jany) understand and use the direction parameter
```

Comment:
Is this comment still relevant?

Reply:
That is done now, comment can be removed.